mlir/include/mlir/Dialect/ArmSME/Transforms/Passes.h

```diff
@@ -24,15 +24,19 @@ namespace arm_sme {
 // the function interface (ABI) and the caller manages PSTATE.SM on entry/exit.
 // In a locally streaming function PSTATE.SM is kept internal and the callee
 // manages it on entry/exit.
-enum class ArmStreaming { Default = 0, Locally = 1 };
+enum class ArmStreamingMode { Default = 0, Locally = 1 };
```

Comment:
Wondering whether this shouldn't be something like:
```cpp
enum class ArmStreamingMode { Disabled = 0, ArmStreaming, ArmLocallyStreaming = 1 };
```
Basically, we'd want this to be consistent with ACLE (which doesn't have `Default`) as well as `ArmZaMode` (which defines `Disabled`).

Reply:
Make sense :+1:
I've removed the "Default" state and added a "Disabled" state. The old "Default" is now"Streaming".

Reply:
Hm, I still see `Default`?